### PR TITLE
add idea: SOB Improve Onboarding

### PR DESCRIPTION
**Pull request by martinklepsch**
Feedback welcome!

### CODE REVIEW

File: ideas/xxx-sob-improve-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+This swarm aims to remove the need for whitelisting users and revamp the onboarding flow at the same time.
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch (10hr for MVP, 30hr+ from MVP onwards)
+- Testing & Evaluation: @Churik
+- UX Contributor: @EugeOrtiz 
+- Dev Contributor: @siphuel or @rcullito
+- Contracts Contributor: @3esmit
+
```
Comment (arash009) [inline ideas/xxx-sob-improve-onboarding.md]: Happy to contribute as well :)

PM Contributor: @arash009

Comment (arash009) [inline ideas/xxx-sob-improve-onboarding.md]: As a general comment is the purpose of this swarm the removal of the whitelisting process for bounty creators, or an overall better process for anyone using the platform?

Are key results 3/4 being tackled in a different swarm?

File: ideas/xxx-sob-improve-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Currently all users wanting to issue a bounty need to go through a whitelisting process. This is required because we currently pay the gas for deploying the bounty contract. In order to make it easier for users to get started we should remove this whitelisting requirement, allowing users to onboard at their own speed.
+
+There are multiple ways the need for whitelisting could be removed:
+
+- Allow (or force) users to pay gas for contract deployment themselves
+- Pay gas for new users but implement some rate-limiting or similar guards to protect from bad actors
+
+These and other alternatives will be evaluated as part of this swarm, whatever decision is taken will be documented in a [Decision Record](https://github.com/status-im/open-bounty/blob/develop/doc/decisions/0001-record-decisions.md). 
+
```
Comment (arash009) [inline ideas/xxx-sob-improve-onboarding.md]: See comment above. I was hoping we would look at this more holistically and ideally identify different use cases as well e.g. (https://github.com/status-im/open-bounty/issues/208)

File: ideas/xxx-sob-improve-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- Design (wireframes) onboarding and bounty-issuing flow that properly accounts for the removed need to get whitelisted and guides users through the creation of their first bounty.
+- Decide on the best way to eliminate whitelisting requirement for immediate future.
+
+**Artifacts:**  
+
+- [ ] mockup of onboarding flow for bounty issuers
+- [ ] mockup of onboarding flow for bounty hunters
+- [ ] Updated welcome email text for both
+- [ ] Decision Record for removal of whitelisting requirement
+
```
Comment (arash009) [inline ideas/xxx-sob-improve-onboarding.md]: "guides users" just refers to visual guides in the workflow I assume?

File: ideas/xxx-sob-improve-onboarding.md (hunk 1 above)
Comment (martinklepsch) [reply ideas/xxx-sob-improve-onboarding.md]: > As a general comment is the purpose of this swarm the removal of the whitelisting process for bounty creators, or an overall better process for anyone using the platform?

Removal of whitelisting requirement and revamped onboarding for anyone using the platform.

> Are key results 3/4 being tackled in a different swarm?

> - KR3: Create FAQs and “How-to” videos & integrate into site/app
> - KR4: Ship funnel metrics for bounty issuers and bounty hunters

Should be tackled in different swarms IMHO. Maybe KR4 could be folded into the metrics one too.

File: ideas/xxx-sob-improve-onboarding.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+**Artifacts:** 
+
+- [ ] Functioning onboarding UI and bounty issuing flow that does not require getting whitelisted.
+- [ ] Emails that are sent to new users are updated
+
+### Iteration 3 — Cleanup `2018-05-30`
+
+Devise steps to fully migrate to new onboarding flow, refactor relevant code, delete code, clean up UI.
+
```
Comment (arash009) [inline ideas/xxx-sob-improve-onboarding.md]: We have a smart contracts contributor defined, I assume there are changes to the multisig as part of this? Is that required to meet the objective of the swarm? Or for something else? Its not defined in any of the iterations?

File: ideas/xxx-sob-improve-onboarding.md (hunk 2 above)
Comment (martinklepsch) [reply ideas/xxx-sob-improve-onboarding.md]: I'd very much like to see https://github.com/status-im/open-bounty/issues/208 being implemented but I think that's different from this idea. Maybe we can integrate a question about the use case described in #208 into the upcoming user-surveys @EugeOrtiz?

File: ideas/xxx-sob-improve-onboarding.md (hunk 3 above)
Comment (martinklepsch) [reply ideas/xxx-sob-improve-onboarding.md]: Well, could be anything UX comes up with. Maybe it's pages, maybe a check list of some kind. Dunno.

**Comment by martinklepsch** [comment]
Initial draft of flows for this revision of onboarding can be found in this [RealtimeBoard](https://realtimeboard.com/app/board/o9J_kzwC3Ag=/)

**Pull request merged**